mantidproject/mslice
# SaveToADS button now enabled in workbench

Changes the check for `in_mantidplot` to include the workbench.

**To test:**



Run MSlice in the workbench - go to the `MD Histo` tab and check the `Save to Mantidplot` button is visible.

Run MSlice in Mantidplot -  go to the `MD Histo` tab and check the `Save to Mantidplot` button is visible.

Run MSlice using the `mslicedevel.sh` script outside of Mantidplot and workbench. Go to the `MD Histo` tab and check the `Save to Mantidplot` button is *not* visible.


Fixes #473
